data/mods/Dark-Skies-Above/monsters/alien_robots.json

```diff
@@ -50,6 +75,7 @@
     "vision_night": 5,
     "special_attacks": [
       [ "TAZER", 12 ],
+      [ "DSA_DRONE_SCAN", 5 ],
```

Comment:
My only concern with giving the seeker the scan is that it's already a somewhat dangerous attacker when angered. That said, it certainly does make them a much more dangerous enemy, so it's just a consideration.

Reply:
Having a drone show up at all should cause an unprepared survivor to run - and they have at least 35 seconds to escape the area without immediate consequences.

I do need to add it to the other drone so they're both worrisome.

Reply:
And I did, it was just a single line special attack so I didn't see it.